aten/src/ATen/native/README.md

```diff
@@ -477,15 +503,26 @@ Here're steps to follow to decide the right dispatch keyword:
 
       Note: current plan on record for ops using this boilerplate is to replace `at::` with `at::native` in
       the implementations and add dispatch section with device keywords instead.
+3. Validate the computed dispatch table matches what you want. You can use `PythonDispatcher` provided in
+[torch/_python_dispatcher.py](https://github.com/pytorch/pytorch/blob/master/torch/_python_dispacher.py).
+It shows for a certain operator, how the computed dispatch table looks like after your registrations.
```

Comment:
Nit: either "how the computed dispatch table looks" or "what the computed dispatch table looks like" 😁 

This is an awesome feature though!